kubernetes/README.md

```diff
@@ -0,0 +1,182 @@
+# Kubernetes
+
+> See the [ci README](../ci/README.md) for the specifics of deploying Kubernetes changes via GitOps.
+
+> :bulb: Both the Google Kubernetes Engine UI and the Lens Kubernetes IDE are useful GUI tools for interacting with a Kubernetes cluster, though you can get by with `kubectl` on the command line.
+
+We deploy our applications and services to a Google Kubernetes Engine cluster. If you are unfamiliar with Kubernetes, we recommend reading through [the official tutorial](https://kubernetes.io/docs/tutorials/kubernetes-basics/) to understand the main components (you do not have to actually perform all the steps).
+
+A [glossary](#Glossary) exists at the end of this document.
+
+## Cluster Administration
+
+We do not currently use Terraform to manage our cluster, nodepools, etc. and major changes to the cluster are unlikely to be necessary, but we do have some bash scripts that can help with tasks such as creating new node pools or creating a test cluster.
+
+First, verify you are logged in and gcloud is pointed at `cal-itp-data-infra` and the `us-west1` region.
+```bash
+gcloud auth list
+gcloud config get-value project
+gcloud config get-value compute/region
+```
+
+### Deploying the cluster
+
+The cluster level configuration parameters are stored in [config-cluster.sh](./gke/config-cluster.sh). Creating the cluster also requires configuring parameters for a node pool named "default-pool" (unconfigurable name defined by GKE) in [kubernetes/gke/config-nodepool.sh](./gke/config-nodepool.sh). Any additional node pools configured in this file are also stood up at cluster creation time.
+
+Once the cluster is created, it can be managed by pointing the `KUBECONFIG`
+environment variable to `kubernetes/gke/kube/admin.yaml`.
+
+```bash
+./kubernetes/gke/cluster-create.sh
```

Comment:
Is this something someone should actually run? May want to distinguish between the stuff on lines 17-20 which seems safe to run just for testing and this seems less safe to just run?